README.md

```diff
@@ -25,18 +25,14 @@ Check out the [examples](examples) directory.
 
 1. Install the library
 
-        git clone https://github.com/jupyter-incubator/sparkmagic
-        cd sparkmagic
-        pip install -e .
+        pip install sparkmagic
 
-2. (Optional) Install the wrapper kernels
+2. (Optional) Install the wrapper kernels. Do `pip show sparkmagic` and it will show the path where `sparkmagic` is installed at. `cd` to that location and do:
 
-        jupyter-kernelspec install remotespark/kernels/sparkkernel
-        jupyter-kernelspec install remotespark/kernels/pysparkkernel
+        jupyter-kernelspec install kernels/sparkkernel
```

Comment:
Also there  are no `kernels/pysparkkernel` or `kernels/sparkkernel` directories when I install from pip.  Please ensure the published package has all the necessary modules.